runtime/src/main/java/org/corfudb/runtime/collections/CorfuStore.java

```diff
@@ -199,26 +200,48 @@ public Query query(@Nonnull final String namespace) {
     }
 
     /**
-     * Subscribe to a specific a table in a namespace or the entire namespace.
-     * Objects returned will honor transactional boundaries
-     * @param streamListener - callback context.
-     * @param namespace - the CorfuStore namespace to subscribe to.
-     * @param tablesOfInterest - If specified, only updates from these tables will be returned.
-     * @param timestamp - If specified, all stream updates from this timestamp will be returned
-     *                  - if null, only future updates will be returned.
+     * Subscribe to transaction updates on specific tables in the namespace.
+     * Objects returned will honor transactional boundaries.
+     *
+     * @param streamListener   callback context
+     * @param namespace        the CorfuStore namespace to subscribe to
+     * @param tablesOfInterest only updates from these tables of interest will be returned
+     * @param timestamp        if specified, all stream updates from this timestamp will be returned
+     *                         if null, only future updates will be returned
      */
+    @Deprecated
     public <K extends Message, V extends Message, M extends Message>
     void subscribe(@Nonnull StreamListener streamListener, @Nonnull String namespace,
                    @Nonnull List<TableSchema<K, V, M>> tablesOfInterest,
                    @Nullable Timestamp timestamp) {
         runtime.getTableRegistry().getStreamManager()
                 .subscribe(streamListener, namespace, tablesOfInterest,
-                (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence());
+                        (timestamp == null) ? getTimestamp().getSequence() : timestamp.getSequence());
+    }
+
+    /**
+     * Subscribe to transaction updates on specific tables with the streamTag in the namespace.
+     * Objects returned will honor transactional boundaries.
+     *
+     * @param streamListener   callback context
+     * @param namespace        the CorfuStore namespace to subscribe to
+     * @param streamTag        only updates of tables with the stream tag will be polled
+     * @param tablesOfInterest only updates from these tables of interest will be returned
```

Comment:
Nit: the updates of tablesOfInterest will not be exactly 'returned', right? Considering that this method returns void, it might help people better understand by not saying 'return'. What about '... will be passed to streamListener'?

Reply:
@Lujie1996 sure, done.